src/current/v25.2/sql-statements.md

```diff
@@ -18,6 +18,7 @@ Statement | Usage
 [`ALTER FUNCTION`]({% link {{ page.version.version }}/alter-function.md %}) | Modify a [user-defined function]({% link {{ page.version.version }}/user-defined-functions.md %}).
 [`ALTER INDEX`]({% link {{ page.version.version }}/alter-index.md %}) | Apply a schema change to an index.
 [`ALTER PARTITION`]({% link {{ page.version.version }}/alter-partition.md %}) | Configure the replication zone for a partition.
+[`ALTER POLICY`]({% link {{ page.version.version }}/alter-policy.md %}) | Alter a [row-level security (RLS)]({% link {{ page.version.version }}/row-level-security.md %}) policy.
```

Comment:
I feel like these statements should go under Data control (basically privileges), not Data definition (schema changes).

Reply:
makes sense to me, moved them all down there and slotted in alphabetically